src/components/CalendarDay.jsx

```diff
@@ -118,12 +116,44 @@ export default class CalendarDay extends React.Component {
       height: daySize - 1,
     };
 
+    const useDefaultCursor =
```

Comment:
As we discussed, maybe worth converting these modifiers to boolean properties or something similar. Might be an easier interface + more intuitive.

Reply:
I think we can do this in a follow-up PR (and one that is slightly less overwhelming to read)